src/components/Assets/SvgImg.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
```

Comment:
Please rename the file to be more specific may be like `SeatIcon` or `SeatComponent` and since this is a react component, move it to folders with react component.